build.js

```diff
@@ -116,9 +116,13 @@ function renderTableHeaderRow(writer, maximumLevel) {
     // SDK columns
     // eslint-disable-next-line no-restricted-syntax
     for (const sdkManifestSuffix of sdkManifests.keys()) {
-      rowWriter.class(`px-1 text-center ${commonCellStyle}`);
+      rowWriter.class(`px-1 align-middle ${commonCellStyle}`);
```

Comment:
You've already got `'align-middle'` in `commonCellStyle`. I think you only need it in one, or the other.

Reply:
Good catch. Done!